src/hotspot/share/opto/superword.cpp

```diff
@@ -2079,27 +2124,39 @@ void SuperWord::schedule() {
     }
 #endif
     _packset.clear();
-    return;
+    return false;
   }
 
-#ifndef PRODUCT
-  if (is_trace_superword_info()) {
-    tty->print_cr("SuperWord::schedule: memops_schedule:");
-    memops_schedule.dump();
-  }
-#endif
+  // (4) Apply the vectorization, including re-ordering the memops.
+  return apply(memops_schedule);
+}
 
+bool SuperWord::apply(Node_List& memops_schedule) {
   CountedLoopNode* cl = lpt()->_head->as_CountedLoop();
-  phase()->C->print_method(PHASE_SUPERWORD1_BEFORE_SCHEDULE, 4, cl);
+  phase()->C->print_method(PHASE_AUTO_VECTORIZATION1_BEFORE_APPLY, 4, cl);
```

Comment:
You could store `phase()->C` into a local variable `C` and use that one.

Reply:
Sure